### DISCUSSION TITLE
IPTV VIDEO CANNOT DEVELOP ON VLC

### DISCUSSION THREAD

**Discussion by Alantienviet** [iptv-org/community#207] [Q&A] [answered]
i have a link this works 100% well...but it can't play on VLC...can someone help me
this is my m3u path but it doesn't work on VLC but works fine on IPTV Lite android

#EXTINF:-1 tvg-id="sctv14hd" group-title="SCTV" "http://vthanh.live/ic/80.png", SCTV14 HD
https://e1.endpoint.cdn.sctvonline.vn/secure/sctv14/sd1/index.m3u8?md5=bs-8dgbSHwp63TDXoCpnkw&expires=2556118740

here is the homepage: https://sctvonline.vn/page/truyen-hinh-ecb1ec92

**Comment by Dum4G** [comment, marked as answer]
Try adding a new line after extinf part #EXTVLCOPT:http-referrer=https://sctvonline.vn/detail/sctv14-789c9f45
You may also want to combine it with something like
#EXTVLCOPT:http-user-agent=Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/91.0.4472.124 Safari/537.36
or
#EXTVLCOPT:http-user-agent=stream

**Reply by Alantienviet** [reply]
i also tried for this line #EXTVLCOPT: http-referrer = https://sctvonline.vn/detail/sctv14-789c9f45 but it still doesn't work...maybe it's because of the md5 prefix after index.m3u8..  . i also tried deleting that md5 line but it broke the link

**Reply by Dum4G** [reply]
Your link is tokenized, that's for sure. If you have tried it in VLC after a good amount of time there's a chance it just expired

**Reply by Alantienviet** [reply]
i'm sure this link is long term...it's been working fine in IPTV Lite for many months...the problem is that IPTV Lite can play this link and VLC can't

**Reply by taodicakhia** [reply]
The link (https://e1.endpoint.cdn.sctvonline.vn/secure/sctv14/sd1/index.m3u8?md5=bs-8dgbSHwp63TDXoCpnkw&expires=2556118740) requires referrer for link and content of link

**Reply by Alantienviet** [reply]
#EXTINF:-1 tvg-id="sctv14hd" group-title="SCTV" tvg-logo="", SCTV14 HD| Kênh Phim Việt
#EXTVLCOPT: http-referrer=https : //sctvonline.vn/detail/sctv14-789c9f45
 https://e1.endpoint.cdn.sctvonline.vn/secure/sctv14/sd1/index.m3u8?md5=bs-8dgbSHwp63TDXoCpnkw&expires=2556118740

i tried this but still cant work on VLC Or IPTV

**Reply by taodicakhia** [reply]
try it by streamlink

**Comment by Sphinxroot** [comment]
https://e1.endpoint.cdn.sctvonline.vn/hls/sctv14/sd1/index.m3u8

https://e1.endpoint.cdn.sctvonline.vn/hls/sctv14/sd2/index.m3u8

Parameter :
strict-origin-when-cross-origin
Access-Control-Allow-Credentials : true   < Login Required to view>
Access-Control-Allow-Headers : *
Access-Control-Allow-Methods : GET, OPTIONS
Access-Control-Allow-Origin : https://sctvonline.vn <access control is to domain only>
Access-Control-Expose-Headers : Content-Length
X-CDN-Edge-Hostname : s1.cdn.sctvonline.vn
Referer : https://sctvonline.vn/
Sec-Fetch-Site : same-site

3 options  

1 SCTV Online android apk   that  use you credentials
2 kodi addon plugin.video.sctv   that  use you credentials
3 another site that restraem the feed , but aint finded one except  x-tream code link
